examples/aws_exocompute/main.go

```diff
@@ -76,25 +79,25 @@ func main() {
 		fmt.Printf("Feature: %v, Regions: %v, Status: %v\n", feature.Name, feature.Regions, feature.Status)
 	}
 
-	// Add exocompute config for the account.
-	exoID, err := awsClient.AddExocomputeConfig(ctx, aws.ID(aws.Default()),
-		aws.Managed("us-east-2", "vpc-4859acb9", []string{"subnet-ea67b67b", "subnet-ea43ec78"}))
+	// Add an exocompute configuration for the account.
+	exoID, err := exoClient.AddAWSConfiguration(ctx, accountID,
+		exocompute.AWSManaged(gqlaws.RegionUsEast2, "vpc-4859acb9", []string{"subnet-ea67b67b", "subnet-ea43ec78"}))
```

Comment:
Is there a reason for using hard-coded VPC and Subnet IDs for AWS?

Reply:
They are made up placeholders for the example code. Anybody wanting to run the code has to put in their own valid values.